backend/app/Http/Controllers/Web/Admin/Connectivity/MqttConnectionController.php

```
@@ -107,46 +123,53 @@ public function show(MachineConnection $mqttConnection)
             ->get();
 
         return Inertia::render('admin/connectivity/mqtt/Show', [
+            'lines' => $this->lineOptions(),
+            'workstations' => Workstation::where('is_active', true)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Workstation::where().


(staticMethod.notFound)